doc/help/CP.md

```diff
@@ -0,0 +1,7 @@
+kdb-cp(1) -- Copy keys within the key database
+================================
+
+## USAGE
+`kdb cp <source> <dest>`  
```

Comment:
You can copy all the keys into a key in the same directory but it won't make a copy of itself below itself. Essentially directories in the Key Database are just keys themselves and can have values? Also do you want me to include any of this in the help text or was this just a learning exercise?